crates/shim/src/cgroup.rs

```diff
@@ -109,19 +109,19 @@ pub fn collect_metrics(pid: u32) -> Result<Metrics> {
         match sub_system {
             Subsystem::CpuAcct(cpuacct_ctr) => {
                 let mut cpu_usage = CPUUsage::new();
-                cpu_usage.set_total(cpuacct_ctr.cpuacct().usage);
+                cpu_usage.total = cpuacct_ctr.cpuacct().usage;
                 let mut cpu_stat = CPUStat::new();
-                cpu_stat.set_usage(cpu_usage);
-                metrics.set_cpu(cpu_stat);
+                cpu_stat.usage = Some(cpu_usage).into();
```

Comment:
Does the new ttrpc not provide setter functions? I think `Some(xxx).into` is a little complex by writing.

Reply:
Protobuf 3 has removed setters/getters for generated structs.
`usage` has [MessageField](https://docs.rs/protobuf/latest/protobuf/struct.MessageField.html) type, which is a wrapper for `Option<Box<T>>` (replacement for `SingularPtrField` used in Protobuf 2). I found `Some().into()` to be the shortest syntax possible here.

Reply:
It's also possible to use `MessageField::some()`, but it looks weird in the context of Rust with std `Option` type. So I'd prefer to use `Some` here.

Reply:
that's OK